linux_os/guide/system/permissions/files/file_permissions_unauthorized_world_writable/bash/shared.sh

```diff
@@ -1,3 +1,3 @@
-# platform = multi_platform_rhel
+# platform = multi_platform_all
 
-(cd / && find . -type f -perm -002 -not -path "./selinux/*" -not -path "./proc/*" -not -path "./sys/*" -not -path "./cgroup/*" -exec chmod -v o-w {} +)
+(cd / && find / -xdev -type f -perm -002 -exec chmod o-w {} \;)
```

Comment:
Seems like having `cd / && find /` is unnecessary. Shouldn't it just be `find / -xdev -type f -perm -002 -exec chmod o-w {} \;` ?

Reply:
Yes, I agree.